internal/core/Cargo.toml

```diff
@@ -73,9 +76,17 @@ unicode-script = { version = "0.5.3", optional = true }
 embedded-graphics = { version = "0.7.1", optional = true }
 integer-sqrt = { version = "0.1.5", optional = true }
 
+image = { version = "0.24.0", optional = true, default-features = false, features = [ "png", "jpeg" ] }
+by_address = { version = "1.0.4", optional = true }
+
+resvg = { version= "0.22", optional = true, default-features = false }
+usvg = { version= "0.22", optional = true, default-features = false, features = ["text"] }
+tiny-skia = { version= "0.6", optional = true, default-features = false }
```

Comment:
I guess that means we can get rid of these dependencies in the gl backend

Reply:
Yes :)